### ISSUE TITLE
LXDE

### ISSUE THREAD

**Issue by layaachmad** [termux/x11-packages#366]
**Package description**
LXDE is good for low end devices it has low resource usage

**Link to home page and sources**
1. Home page: http://www.lxde.org/
2. Source code: https://github.com/lxde

**Comment by leapofazzam123** [comment]
It's outdated and replaced with LXQt

**Comment by Yisus7u7** [comment]
It's very true, but there are packages from that desktop that would be very good if they were ported, for example, lxapariencie to combine it with openbox and other things

**Comment by leapofazzam123** [comment]
> there are packages from that desktop that would be very good if they were ported, for example, lxapariencie to combine it with openbox and other things

Yeah, i guess

**Comment by Yisus7u7** [comment]
Yes, for those who like desktop ricing, lxapariencie use it a lot in tutorials jajajaja

**Comment by Yisus7u7** [comment]
@suhan-paradkar Sorry to bother you, I know you are working with mate, but could I compile to lxde? I was analyzing it and it does not look difficult to port the termux, if it were not for intool I would compile it without problems, but as in termux it is not intool

**Comment by Yisus7u7** [comment]
but I was analyzing lxde and it would be the perfect gtk alternative to xfce, it is very light and consumes very few resources, apart from it it has useful things, and just by changing the gtk theme it can look modern

**Comment by Yisus7u7** [comment]
also the lxaparience application is very simple, and useful to combine them with wm such as openbox, bspwm or fluxbox, I see that this app is widely used for desktop ricing with WMs.

**Comment by Yisus7u7** [comment]
@WMCB-Tech what's your opinion about it

**Comment by WMCB-Tech** [comment]
I don't use LXDE but there's already LXQT, though would maintain other LX* desktop ports

I find LXQt is enough for me thus it's actively maintained so that DE is enough

*Unless someone is willing to port it, so why not*

**Comment by Yisus7u7** [comment]
@WMCB-Tech Lxde is lightweight, but its dependencies are not, Installing lxqt on termux weighs more than installing xfce4-*, all because of its dependencies

**Comment by Yisus7u7** [comment]
But I can not deny that lxqt consumes very few resources, just as lxde would be very lijero to have it in termux, it does not require so many dependencies, and its optimization is similar to that of lxqt

**Comment by suhan-paradkar** [comment]
@Yisus7u7 go ahead.... You don't need to ask permissions....
But please begin with a fresh fork....
Your current fork is messed up.. if you have no PRs pending.. then delete your current fork and fork it again.. and please.. donot use the master branch.. better create a separate branch for a separate PR

**Comment by Yisus7u7** [comment]
Ok, tanks!

**Comment by Yisus7u7** [comment]
![Screenshot_20210714-153846](https://user-images.githubusercontent.com/64093255/125700150-b0a1e7e7-93e2-47dc-85a3-0204c7c6b713.png)
![Screenshot_20210714-150925](https://user-images.githubusercontent.com/64093255/125700155-5279b292-2c6c-4214-baa1-7f7d58b8f866.png)
Stonks!

**Comment by Yisus7u7** [comment]
#382 
#383

**Comment by Yisus7u7** [comment]
@layaachmad Do not close the problem, some dependencies are still missing

**Comment by Yisus7u7** [comment]
This hasn't ended yet

**Comment by layaachmad** [comment]
And how about lxde-panel and lxsession

**Comment by Yisus7u7** [comment]
I'm having trouble compiling it

@suhan-paradkar Could you take care of those dependencies?

**Comment by suhan-paradkar** [comment]
I am pretty busy right now.. but sure... I can try...

**Comment by Yisus7u7** [comment]
I would appreciate it, I am very anxious also to have lxde in termux

**Comment by Yisus7u7** [comment]
How is everything going?
